lib/game_box_web/live/arena_live.ex

```diff
@@ -47,20 +47,28 @@ defmodule GameBoxWeb.ArenaLive do
 
     ~H"""
     <%= if board == "" do %>
-      <.h5 class="text-center" label="Arena" />
-      <.h1 class="text-center" label={@arena.arena_id} />
-
-      <%= if is_nil(@game_selected) do %>
-        <%= if @is_host do %>
-          <.p class="text-center">Select a game to get started!</.p>
-        <% else %>
-          <.p class="text-center">Waiting for the host to select a game...</.p>
+      <div class="my-24">
+        <.h5 class="text-center !text-primary" label="Arena" />
+        <.h1 class="text-center" label={@arena.arena_id} />
+
+        <%= if is_nil(@game_selected) do %>
+          <%= if @is_host do %>
+            <.p class="text-center">Select a game to get started!</.p>
+          <% else %>
+            <.p class="text-center">Waiting for the host to select a game...</.p>
+          <% end %>
         <% end %>
-
-        <div class="grid grid-cols-3 gap-x-12 gap-y-12 mb-12">
+      </div>
+      <%= if is_nil(@game_selected) do %>
+        <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-x-12 gap-y-12 mb-12">
           <%= for game <- @games do %>
             <.card>
-              <.card_media src={game.artwork} />
+              <.card_media :if={game.artwork} src={game.artwork} />
+              <.card_media
```

Comment:
Do you think it makes sense to encapsulate this default image logic into the card_media component?

Reply:
I thought about it, but we're also using this card component for other layout elements where we don't want media at all. I could maybe make it an optional slot based on something like `always_show_media` being set to true per card (and having it set to false by default)... what do you think?

Reply:
if it has other use cases without it i think its fine to leave it this way